WordPress/Classes/ViewRelated/Blog/QuickStartChecklistViewController.swift

```diff
@@ -167,7 +171,11 @@ private class QuickStartChecklistDataSource: NSObject, UITableViewDataSource {
 
         switch section {
         case .congratulations:
-            return shouldShowCongratulations() ? 1 : 0
```

Comment:
Noting we don't really need this change now if you prefer the original formatting. :)